### ISSUE TITLE
Unknown error when Tapping

### ISSUE THREAD

**Issue by Mjstq** [shamhi/TapSwapBot#187]
2024-07-20 20:46:39 | ERROR    | 178 - session001 | Unknown error when Tapping: 400, message='Bad Request', url=URL('https://api.tapswap.club/api/player/subm
it_taps') | Response text: {"statusCode":400,"message":"invalid_request[2]"}...

**Comment by supmyhava** [comment]
i think there is a problem when you have more than 1 sessions
so just create 1 session 1 bot

**Comment by s1rne** [comment]
#185

**Comment by danazor** [comment]
this bot is working? i tried a few weeks ago and cant run

**Comment by s1rne** [comment]
> this bot is working? i tried a few weeks ago and cant run @s1rne

Yes, Recently, there was a problem with multi-accounts, but I wrote a crutch solution
